python/hyperon/atoms.py

```diff
@@ -2,6 +2,31 @@
 from hyperonpy import AtomKind
 from typing import Union
 
+
+class Char:
```

Comment:
I would move this class into `stdlib.py` because it is not a part of a core library but a part of the MeTTa `stdlib` instead.

Reply:
Yes, I don't think we need separate file, it is a tiny class.